setup.py

```diff
@@ -56,6 +55,11 @@
     ),
     install_requires=REQUIREMENTS,
     include_package_data=True,
-    python_requires=">=3.6",
+    python_requires=">=3.7",
+    project_urls={
+        "Bug Tracker": "https://github.com/Qiskit-Extensions/qiskit-cold-atom/issues",
+        "Documentation": "https://qiskit.org/documentation/",
```

Comment:
This should maybe be replaced by the link specifically to the cold atom documentation once we have that